build.fsx

```diff
@@ -95,7 +99,14 @@ Target "Build" (fun _ ->
     |> Log "AppBuild-Output: "
 )
 
-// Build lots of packages for differet VM sizes
+let filesForZip =
```

Comment:
I don't think there is, we could scrape the bin directory but we'd end up with assemblies which aren't really necessary like all of the Azure management libraries.